### PR TITLE
sign up, sign in , edit user

### PR DESCRIPTION
**Pull request by phornsopheak**

### CODE REVIEW

**Comment by RathanakSreang** [comment]
Ok

**Comment by HorNorin** [comment]
OK

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<h1><%= t "menus.edit" %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
```
Comment (RathanakSreang) [inline app/views/devise/registrations/edit.html.erb]: `<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: {method: :put}) do |f| %>`

**Comment by RathanakSreang** [comment]
Ok

**Pull request merged**